R/putMadratMessage.R

```diff
@@ -0,0 +1,39 @@
+#' Tool: putMadratMessage
+#'
+#' Store a madrat message in the madrat environment. The madrat environment
+#' behaves similar like global options, except that 1) messages will also
+#' be stored in cache files and restored when a cache file is being loaded and
+#' 2) messages are always stored in lists with messages split by function calls
+#' where the message was triggered.
+#'
+#' @param name The category in which the message should be stored
```

Comment:
What categories do you mean here? The example puts "test" here, but I'm still not sure how to use this. It is an arbitrary string chosen by the user?